src/pages/Login/SignUp.js

```diff
@@ -0,0 +1,208 @@
+import React, { Component } from 'react';
+import './SignUp.scss';
+
+class SignUp extends Component {
+  constructor() {
+    super();
+    this.state = {
+      username: '',
+      password: '',
+      name: '',
+      nickname: '',
+      email: '',
+      phone_number: '',
+      address: '',
+    };
+  }
+
+  handleInput = e => {
+    const { name, value } = e.target;
+    this.setState({ [name]: value });
+  };
+
+  handleSignUp = () => {
+    fetch('http://10.58.2.67:8000/users/signup', {
+      method: 'POST',
+      body: JSON.stringify({
+        username: this.state.username,
+        password: this.state.password,
+        name: this.state.name,
+        nickname: this.state.nickname,
+        email: this.state.email,
+        phone_number: this.state.phone_number,
+        address: this.state.address,
+      }),
```

Comment:
```suggestion
      body: JSON.stringify({
        ...this.state
      }),
```
로 표현해줘도 될 것 같습니다. 스프레드 연산자 + 객체 단축 속성명을 이용한 동작입니다!